### PR TITLE
Selase & Simon: Pokemon

### PR DESCRIPTION
**Pull request by BB-Simon**
- **A home page showing a list of items that you can "like."**
- **A popup window with more data about an item that you can use to comment on it.**
- **Pokemos APIs**
- **Get a list of items with a unique item id (or generate the unique id).**
- **For a given item, get detailed information about it.**
- **The Involvement API to show the item likes**
- **When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated**
- **When the user clicks on the "Comments" button, the Comments popup appears**
- **When the popup loads, the webapp retrieves data from:**
    - **The selected API and shows details about the selected item.**
    - **The Involvement API to show the item comments.**
- **Counters**
    - **The number of items (home).**
    - **The number of comments (comments popup).**

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @dselasea, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of Gitflow. ✔️ 
 - Great UI design. ✔️ 
 - All API calls working as expected. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/commentCounter.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    const comments = document.getElementById('comment-list');
+    const countElements = commentCounter(comments);
+    expect(countElements).not.toBeNull();
+    expect(countElements).not.toBeUndefined();
+    expect(countElements).not.toBeFalsy();
+    expect(countElements).not.toBeGreaterThan(4);
+    expect(countElements).not.toBeLessThan(4);
+    expect(countElements).toBe(4);
+    expect(countElements).toEqual(4);
+  });
```
[the remark this answers is not in the document]
Comment (Abenezer-Tilahun) [reply src/__tests__/commentCounter.test.js:26]: Fixed.

File: src/__tests__/itemCounter.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    const count = itemCounter(pokemonContainer);
+    expect(count).not.toBeNull();
+    expect(count).not.toBeUndefined();
+    expect(count).not.toBeFalsy();
+    expect(count).not.toBeGreaterThan(3);
+    expect(count).not.toBeLessThan(3);
+    expect(count).toBe(3);
+    expect(count).toEqual(3);
+  });
+});
```
[the remark this answers is not in the document]
Comment (Abenezer-Tilahun) [reply src/__tests__/itemCounter.test.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @BB-Simon @dselasea 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**